attributes/network/1000_abci_backend.adoc

```diff
@@ -16,5 +16,52 @@ It exposes the following endpoints:
 - `abci.endBlock`.
 - `abci.init`.
 - `abci.info`.
+- `abci.checkTx`.
+    Method controls what transactions get added to a new block
+- `abci.manyStatus`.
```

Comment:
We should have a separate attribute for MANY-specific network information. That might require a bit more design and thinking (and look into the Rosetta API).